specs/agents/tracing-spans.md

```diff
@@ -17,6 +17,8 @@ for simpler and more performant UI queries.
 Each transaction has a `type` field, each span has both `type` and `subtype` fields.
 The values for each of those fields is protocol-specific and defined in the respective instrumentation specification
 for each protocol.
+If no `span.type` is provided or the value is an empty string, the agent needs to set a default value.
+APM agents can either derive a default value based on the span context, or fall back to the generic default value `custom`.
```

Comment:
What's the logic for deriving the default value based on the span context?

Reply:
Probably not worth defining this, as it should be an edge case that users define an empty string. Will update accordingly.